@here/harp-mapview/test/TextElementsRendererTest.ts

```diff
@@ -599,7 +618,7 @@ const tests: TestCase[] = [
     }
 ];
 
-describe("TextElementsRenderer", function() {
+describe.only("TextElementsRenderer", function() {
```

Comment:
you missed undoing this, I guess you did it to test locally.

Reply:
Right, thanks for pointing it out, thats probably why all the travis CI complained, as it was flaky recently i didnt even check.